index.html

```diff
@@ -265,8 +268,8 @@ <h3 class="font-bold font-body text-2xl md:text-3xl mb-4 leading-tight">Hosted,
             <div class="md:w-5/12 lg:w-6/12 px-6 sm:px-10 md:px-16 lg:px-24 xl:px-32 md:pr-0 lg:pr-0 xl:pr-0">
               <h3 class="font-bold font-body text-2xl md:text-3xl mb-4 leading-tight">Elastic Scale</h3>
               <div>
-                <p class="my-4">Flyte is purpose built to scale. With a fully distributed, fault-tolerant control plane, we have no single point of failure and can scale to multiple clusters, thousands of nodes, and thousands of concurrent workflows.</p>
-                <p class="my-4">Our scale is proven at Lyft, where Flyte has been serving production model training and data processing for over three years, becoming the de-facto platform for teams like Pricing, Locations, ETA, Mapping, Autonomous, and more. In fact, Flyte manages over 7,000 unique workflows at Lyft, totaling over 100,000 executions every month, 1 million tasks, and 10 million containers.</p>
+                <p class="my-4">Flyte was born out of the need to scale to multiple tenants, yet be intuitive and simple to use. It is built ground up to be distributed, fault-tolerant control plane, with no single point of failure. It can scale to multiple clusters, thousands of nodes, and thousands of concurrent workflows.</p>
```

Comment:
```suggestion
                <p class="my-4">Flyte was born out of the need to scale to multiple tenants, yet be intuitive and simple to use. It is built from the ground up to be a distributed, fault-tolerant control plane, with no single point of failure. It can scale to multiple clusters, thousands of nodes, and thousands of concurrent workflows.</p>
```